templates/paragraphs/paragraph--oe-illustration-list-images.html.twig

```diff
@@ -0,0 +1,20 @@
+{#
+/**
+ * @file
+ * Theme override to display the 'Illustration list with images' type paragraph.
+ *
+ * @see ./modules/contrib/paragraphs/templates/paragraph.html.twig
+ */
+#}
+{% if title is not empty %}
+  <h2 class="ecl-u-type-heading-2">
+    {{ title }}
+  </h2>
+{% endif %}
+
+{{ pattern('list_with_illustration', {
+  'column': column,
+  'zebra': zebra,
+  'items': items,
+  'squared': squared
```

Comment:
Please remove any reference in our code to "squared", it's an incorrect word as I wrote in the tickets. This should be either "square_image", or "image_ratio" with options "default" or "square".